<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>-51</x>
         <y>-38</y>
         <width>1624</width>
         <height>824</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>1600</width>
            <height>800</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>120</y>
             <width>213</width>
             <height>138</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="verticalSpacing">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>x0:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>y0:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="text">
               <string>1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="text">
               <string>1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>N:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_4">
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_plot">
              <property name="text">
               <string>Update the solution chart</string>
              </property>
              <property name="iconSize">
               <size>
                <width>300</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="SolutionsWidget" name="SolutionsWidget" native="true">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>20</y>
             <width>1131</width>
             <height>731</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>10</height>
            </size>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>310</y>
             <width>194</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Show Euler Method solution</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>280</y>
             <width>181</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Show analytical solution</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget_2">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>420</y>
             <width>211</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="verticalSpacing">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>n0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_GE_from">
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>N</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_GE_to">
              <property name="text">
               <string>100</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_global_error">
              <property name="text">
               <string>Update the error chart</string>
              </property>
              <property name="iconSize">
               <size>
                <width>300</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionas">
   <property name="text">
    <string>as</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SolutionsWidget</class>
   <extends>QWidget</extends>
   <header>solutionswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
